### PR TITLE
Add compilation GH action

### PR DESCRIPTION
**Pull request by lzap**
This is an attempt to write a compilation GH action on ubuntu OS. Wish me luck.

### CODE REVIEW

**Comment by ekohl** [comment]
I'd suggest to copy https://github.com/pulp/pulpcore-selinux/commit/41d4bba87fcbb174f7c24dccce7e8891b5856d66 instead so you actually compile on EL7 and EL8.

**Comment by lzap** [comment]
Nice! Will do a followup PR.

**Comment by lzap** [comment]
https://github.com/theforeman/foreman-selinux/pull/117

File: .github/workflows/compile.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install SELinux
+        run: sudo apt-get install policycoreutils selinux-utils selinux-basics
+
+      - name: Compile policy for RHEL7
+        run: make DISTO=rhel7
```
Comment (adamruzicka) [inline .github/workflows/compile.yaml:17]: Shouldn't it be `DISTRO`?

Comment (lzap) [reply .github/workflows/compile.yaml:17]: Yes fixed in the other PR.

**Comment by lzap** [comment]
FYI I had to merge this otherwise GH Actions won't trigger. I could not tell if it was working at all.

**Comment by ekohl** [comment]
> FYI I had to merge this otherwise GH Actions won't trigger. I could not tell if it was working at all.

What I usually do is to also add the `push` action and check how it runs on my own fork.

**Pull request merged**